source/mongodb/internal-database-usage.txt

```diff
@@ -70,6 +78,14 @@ App Services stores a information about unsyncable documents in the
    You can read from the ``__realm_sync.unsynced_documents`` collection,
    but you should not modify it in any way.
 
+If Device Sync is paused due to hitting the 100,000 unsyncable document limit,
```

Comment:
If we're adding this here, we should probably remove the "You need to manually resolve this by..." at line 70 above.